dotnet/agent-nr-config/Dockerfile

```diff
@@ -0,0 +1,44 @@
+# syntax=docker/dockerfile:1
+FROM mcr.microsoft.com/dotnet/sdk:6.0 AS build
+# Download and unpack the agent
+# Refer to https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/releases for the available releases.
+RUN apt-get update && apt-get install unzip \
+	&& wget "https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/releases/download/v0.1.0-beta.1/opentelemetry-dotnet-instrumentation-linux-glibc.zip" -O /tracer-home.zip \
+	&& unzip -d /tracer-home /tracer-home.zip
+WORKDIR /src
+
+COPY ["agent-nr-config.csproj", "."]
+RUN dotnet restore "./agent-nr-config.csproj"
+
+# Build the application in a framework dependent mode.
+# Automatic instrumentation is not currently supported for self-contained deployments or builds.
+COPY . .
+RUN dotnet build "agent-nr-config.csproj" -c Release -o /app/build
+
+FROM mcr.microsoft.com/dotnet/aspnet:6.0 AS final
+WORKDIR /app
+# Copy the built application
+COPY --from=build /app/build .
+# Install the agent
+COPY --from=build /tracer-home /tracer-home
+# This env variable points to the installation directory for the agent
+ENV OTEL_DOTNET_AUTO_HOME="/tracer-home"
+ENV OTEL_PROFILER_EXCLUDE_PROCESSES="dotnet,dotnet.exe"
+ENV DOTNET_STARTUP_HOOKS="/tracer-home/netcoreapp3.1/OpenTelemetry.AutoInstrumentation.StartupHook.dll"
+ENV DOTNET_ADDITIONAL_DEPS="/tracer-home/AdditionalDeps"
+ENV DOTNET_SHARED_STORE="/tracer-home/store"
+ENV OTEL_DOTNET_AUTO_ENABLED_INSTRUMENTATIONS="AspNet"
+ENV OTEL_TRACES_EXPORTER="otlp"
```

Comment:
Are these env vars all standard?  Might be good to link to the auto-instrumentation docs that detail what each one is doing.